src/StreamJsonRpc.Shared/JsonRpc.cs

```diff
@@ -80,12 +85,13 @@ internal OutstandingCallData(object taskCompletionSource, Action<JsonRpcMessage>
         /// </summary>
         /// <param name="stream">A bidirectional stream to send and receive RPC messages on.</param>
         /// <param name="target">An optional target object to invoke when incoming RPC requests arrive.</param>
+        /// <param name="passParameterAsObject">Value which indicates whether there can be only one parameter and the parameter must be passed as an object</param>
         /// <returns>The initialized and listening <see cref="JsonRpc"/> object.</returns>
-        public static JsonRpc Attach(Stream stream, object target = null)
+        public static JsonRpc Attach(Stream stream, object target = null, bool passParameterAsObject = false)
```

Comment:
FYI I completed that PR and re-built your PR in appveyor. It now registers as a build failure due to the breaking API changes.